example/datasets/create_model_db.sql

```diff
@@ -1 +0,0 @@
-CREATE DATABASE IF NOT EXISTS sqlflow_models;
```

Comment:
This `.sql` is needed for SQLFlow quickstart.

Reply:
Done.